<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaInicioLocacao</class>
 <widget class="QDialog" name="TelaInicioLocacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Voltar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LOCAÇÃO DE VEICULO</string>
   </property>
  </widget>
  <widget class="QLabel" name="idCli">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>41</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="idMotoris">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>41</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>moto</string>
   </property>
  </widget>
  <widget class="QLabel" name="idVeicu">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>41</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>veicu</string>
   </property>
  </widget>
  <widget class="QLabel" name="idCat">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>41</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>Cat</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>111</y>
     <width>661</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>CLIENTE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbmCliente"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>MOTORISTA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbmMotorista"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>VEICULO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbmVeiculo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_Categoria">
        <property name="text">
         <string>CATEGORIA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbmCategoria"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>DATA DE RETIRADA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="txtDataRetirada"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>DATA DE DEVOLUÇÃO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="txtDataDevol"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>QUILOMETRAGEM INICIAL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtQuiInicial"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>QUILOMETRAGEM FINAL </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtQuilFinal"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>VALOR DO CAUÇÃO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCauc"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>VALOR DO SEGURO </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSeguro"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>PAGAMENTO FINAL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPagamentoFinal"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>601</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Fazer Locação</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Fazer Devolução</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Fazer Reserva</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_CadastrarMF">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>650</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar Motorista F</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
